README.md

```diff
@@ -299,6 +299,39 @@ const MyStyledDiv = glamorous.div(
 <MyStyledDiv /> // styles applied: {padding-top: 1, padding-right: 2, padding-bottom: 3, padding-left: 4} and anything coming from `extra-thing`.
 ```
 
+##### withProps
```

Comment:
I think supporting both would be convenient - would your concern in https://github.com/paypal/glamorous/issues/145#issuecomment-304471670 apply here though?